src/rpc/blockchain.cpp

```diff
@@ -803,6 +757,58 @@ static RPCHelpMan getmempoolentry()
     };
 }
 
+static RPCHelpMan getblockfrompeer()
+{
+    return RPCHelpMan{"getblockfrompeer",
+                "\nAttempt to fetch block from a given peer.\n"
```

Comment:
Should also mention that this will disconnect the peer if the peer pretends to not have the block

Reply:
That seems a bit too deep in the net_processing weeds?